lib/actions-oclif/version.ts

```diff
@@ -0,0 +1,79 @@
+/**
+ * @license
+ * Copyright 2019 Balena Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Command, flags } from '@oclif/command';
+import { stripIndent } from 'common-tags';
+
+interface FlagsDef {
+	all?: boolean;
+	json?: boolean;
+}
+
+export default class VersionCmd extends Command {
+	public static description = stripIndent`
+		Display version information for the balena CLI and/or Node.js.
+
+		Display version information for the balena CLI and/or Node.js.
+		If you intend to parse the output, please use the -j option for
+		JSON output, as its format is more stable.
+`;
+	public static examples = [
+		'$ balena version',
+		'$ balena version -a',
+		'$ balena version -j',
+	];
+
+	public static usage = 'version';
```

Comment:
If the assignment to `usage` above was deleted, `balena version --help` would print the exact same output: the default generated by oclif. However, the static value is used to generated the `cli.markdown` document, by code in `automation/markdown.ts`.